references/cli/status-ui/model/info.go

```diff
@@ -0,0 +1,67 @@
+package model
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/oam-dev/kubevela/apis/types"
+	"github.com/oam-dev/kubevela/pkg/utils/helm"
+	"github.com/oam-dev/kubevela/version"
+)
+
+type Info struct {
+	cluster string
+}
+
+const (
+	UNKOWN_VERSION = "UNKNOWN"
```

Comment:
By convention, we typically use MixedCase rather than ALL_CAPS, even if it is a const.

Reply:
ok thanks, i have fixed it.